### PR TITLE
CODEOWNERS: Add codeowners for loki operator

### PR DESCRIPTION
**Pull request by kavirajk**
**What this PR does / why we need it**:
Explicit code-owners for loki operator

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)

### CODE REVIEW

**Review by dannykopping — APPROVED** [review]

File: CODEOWNERS
Comment (periklis) [inline CODEOWNERS:8]: Can you please add my co-maintainer having merging rights @xperimental here please.

**Review by periklis — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (kavirajk) [reply CODEOWNERS]: done.

**Comment by kavirajk** [comment]
@periklis it will get merged once you approve.

**Review by periklis — APPROVED** [review]

**Pull request merged**
